tests/test_dimension.py

```diff
@@ -1515,7 +1515,7 @@ def test_issue_1927(self, factor):
 
         op = Operator(Eq(f, 1))
 
-        assert op.arguments()['time_M'] == 4*(save-1)  # == min legal endpoint
+        assert op.arguments()['time_M'] == 4*save-1  # == min legal endpoint
```

Comment:
I had a comment here which appearly I forgot to fire up...

anyway: was my old comment (min legal endpoint) wrong? or is the new one to be updated?

Reply:
The comment is correct, the value wasn't